src/api/auth.ts

```diff
@@ -87,6 +88,8 @@ async function findOrCreateUser(clientId: string, email: string): Promise<User>
     const isAdmin = apiEnv.ASSIGN_ADMIN_ROLE_ON_SIGN_UP.includes(email);
     const roles: [Role] = [isAdmin ? 'ADMIN' : 'VISITOR'];
     console.log(`Creating user ${email} roles ${roles} id ${clientId}`);
+    // using sendgrid to notify admin a new user is created
```

Comment:
This comment doesn't add value because the code is already self-explanatory (which is ideal).